.github/workflows/basic-tests.yml

```diff
@@ -75,3 +75,23 @@ jobs:
         BRANCH: gh-pages
         FOLDER: _gh-pages
         SINGLE_COMMIT: true
+  deploy_documentation_release:
```

Comment:
@GarethCabournDavies Doesn't this have code duplication here?

Reply:
To further explain, it looks like the documentation release is now duplicated rather than simply running on both master and the tag version.

Reply:
yeah, I couldnt work out how to move to `latest _gh-pages/${GITHUB_REF#refs/tags/}` or `current` (or whatever) dynamically

Reply:
@GarethCabournDavies I don't really follow. Wouldn't you just have an if statement in the yaml in the case of a release and then essentially do what you do here, except without all the duplicated bits?

Reply:
Ah, I think I didn't realise you can effectively use shell if statements in the yaml, I will try deduplicating